meinberlin/apps/plans/forms.py

```diff
@@ -58,10 +58,8 @@ class Meta:
         required_for_project_publish = ['plans']
 
     def save(self, commit=False):
-        self.instance.plans.clear()
-        for plan in self.data['plans']:
-            self.instance.plans.add(plan)
-        self.instance.save()
+        plans = self.data['plans']
```

Comment:
Aha! If you review my stuff, I like when you tell me how to do it properly and then I have to do it myself and learn how to, so that you don't have to tell me again next time. :) But I guess, not everyone likes that.

Reply:
Doch, we all like it. Than as we discussed we do pull request the "propper" way and the reviewer just requests changes.